http4k-testing-chaos/src/main/kotlin/org/http4k/chaos/ChaosFilters.kt

```diff
@@ -0,0 +1,34 @@
+package org.http4k.chaos
+
+import org.http4k.core.Filter
+
+const val LATENCY_HEADER = "X-Chaos-Behaviour"
+
+object ChaosFilters {
+    object ChaosPreFilter {
```

Comment:
Can probably combine these into a single filter and then have 2 methods on the Policy shouldInject(Request) and shouldInject(Response) - both of which can default to false.